css/styles.css

```diff
@@ -0,0 +1,58 @@
+header {
+  background-color: black;
+  font-family: 'Monster Hunter', sans-serif;
+}
+
+.container {
+  background-color: gray;
+}
+
+.title-row {
+  text-align: center;
+  font-family: 'Monster Hunter', sans-serif;
+  color: #94ca76;
+}
+
+.filter {
+  text-align: right;
+  font-size: xx-large;
+  margin-right: 30px;
+  color: black;
+}
+
+.row-head {
+  color: white;
+}
+
+.row-head h1 {
+  margin-left: 15px;
+  margin-right: 2px;
+  margin-top: 30px;
+}
+
+.row-head h2 {
+  margin-left: 75px;
+  margin-right: 2px;
+  margin-top: 35px;
+}
```

Comment:
This is rather awkward using hard set amounts of margin to space these. I think this would be better done if you use `flexbox`. You can have the parent container be a `flex` container and have 2 children. First is the Title, second is another flex container with the icons.

```
// justify-between would apply justify-content: space-between to
// move icons to right side of nav bar
<header class="row justify-between">
  <h1 class="selected">Monsters</h1>
  <div class="row">
     <i class="fa-solid fa-star not-selected"></i>
     <i class="fa-solid fa-dice not-selected"></i>
  </div>
</header>
```